src/Preview.tsx

```diff
@@ -14,13 +14,16 @@ import { warning } from 'rc-util/lib/warning';
 import useFrameSetState from './hooks/useFrameSetState';
 import getFixScaleEleTransPosition from './getFixScaleEleTransPosition';
 import { context } from './PreviewGroup';
+import { ImageStatus } from './Image';
 
 const { useState, useEffect } = React;
 
 interface PreviewProps extends Omit<IDialogPropTypes, 'onClose'> {
   onClose?: (e: React.SyntheticEvent<Element>) => void;
   src?: string;
   alt?: string;
+  /** thumbnail src value */
+  thumbnail?: string;
```

Comment:
If Preview use `thumbnail` too weird

https://user-images.githubusercontent.com/14831261/103391539-f4b87f00-4b54-11eb-9e95-493e8f19d573.mp4

Reply:
😅 Maybe my thumbnails are a bit extreme.

Currently, the replacement is done when the image is loaded to achieve fast switching.
And thumbnail will appear on the first load and before the load is completed. After that it won't even appear.

Reply:
Yep, but is not universal, load status maybe `Image` `<Spin />` or some text, like `placeholder` prop

Reply:
🤔 expose the loading state? let me try

Reply:
I mean this pr resolve Image src not same as preview src, ant other issues like Preview placeholder(if loading) we maybe create new or and Image placeholder is enough

Reply:
understand 👌
Not a good way. This PR will not process `thumbnails` / `load state`.
Hope to export `preview-src`.

eg: aliyun oss
```tsx
<Image
  src="https://xxx.xx/xxx.png?x-oss-process=image/auto-orient,1/resize,p_10/quality,q_10"
  preview={{
    src: 'https://xxx.xx/xxx.png',
  }}
/>;
```
Thanks 🙆‍♂️